package/autoyast2.spec

```diff
@@ -281,6 +281,11 @@ done
 %dir %{yast_libdir}/autoinstall/dialogs
 %{yast_libdir}/autoinstall/dialogs/*.rb
 
+%dir %{yast_libdir}/autoinstall/widgets
```

Comment:
you can do it as oneliner by simple `%{yast_libdir}/autoinstall/widget` this adds dir and all content. This way make sense only to catch if some adds accidentally to install some non ruby files.